### ISSUE TITLE
Support for 3rd party one-step-checkout with custom checkout url

### ISSUE THREAD

**Issue by HansiBlomberg** [magesuite/magepack#22]
"It would be nice if" a future version would support an option for custom checkout url. My store use a third party one-step-checkout module and the checkout url can be set to any url by the administrator.

**Comment by roma-glushko** [comment]
@HansiBlomberg seems to be related to https://github.com/magesuite/magepack/issues/21

**Comment by HansiBlomberg** [comment]
> @HansiBlomberg seems to be related to #21

Yes indeed. Took a quick peek at the source code, does not look too complicated so I will probably just modify it myself to cover my situation.
